_schemaV2.graphql

```diff
@@ -7074,6 +7074,20 @@ type Lot implements Node {
   slug: ID!
 }
 
+type Lot2 {
```

Comment:
I don't want to get too hung up on naming decisions, but can you talk a little about Lot vs Lot2 and MyBids vs MyBids2? I think my biggest concern is that there's not a lot of contextual information of why this exists and how it differentiates.

Reply:
Oh yeah! i actually need to update these before merge to something more reasonable. (Can explain a bit more tomorrow.)

Reply:
Updated to `CausalityLotState` per [this slack convo](https://artsy.slack.com/archives/C01PBEBRB0V/p1616105736019600)